### PR TITLE
SAM-2794: a few small issues with samigo email notifications

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAM-2794

address the following minor issues:

* Catch possible NPE/NFE when accessing assessment setting for instructor notification
* Remove hard-coded Dayton instance name in favour of "ui.service" from sakai.properties
* Remove hard-coded en_US locale from email template XML so the templates are loaded as 'default' locale

Also did some minor clean-up in the files while I was editing them.

### CODE REVIEW

**Comment by ottenhoff** [comment]
Okay changes seem good

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java [hunk 1]
```diff
@@ -353,7 +361,14 @@ public AssessmentFacade save(AssessmentSettingsBean assessmentSettings, boolean
...
     assessment.updateAssessmentMetaData(SecureDeliveryServiceAPI.TITLE_DECORATION, titleDecoration );
 
-    assessment.setInstructorNotification(Integer.valueOf(assessmentSettings.getInstructorNotification()));
+    try
+    {
+        assessment.setInstructorNotification(Integer.valueOf(assessmentSettings.getInstructorNotification()));
+    }
+    catch( NullPointerException | NumberFormatException ex )
+    {
+        LOG.warn( ex );
```
Comment (master-bob) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java]: In the case of a NPE or NFE, should the assessment Instructor Notification be set to the [default](https://github.com/sakaiproject/sakai/blob/4d16ce1a21aa50e4335d5a64d00660a6f86c7f0b/samigo/samigo-api/src/java/org/sakaiproject/samigo/util/SamigoConstants.java#L57)?

Comment (bjones86) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java]: Yes, that's a good point, thanks @master-bob. I'll whip something up.

Comment (bjones86) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java]: I've opened #2099 to address this

**Pull request merged**